t/test_uganda.rb

```diff
@@ -0,0 +1,44 @@
+
+require 'csv_to_popolo'
+require 'minitest/autorun'
+require 'pry'
+require 'json'
+
+
+describe 'monaco' do
+  subject { Popolo::CSV.new('t/data/uganda.csv') }
+
+  def memberships(id) 
+    subject.data[:memberships].find_all { |m| m[:person_id] == id } 
+  end
+
+  it 'should have no Post for a normal membership' do
+    alero = memberships('mp_277').first
+    alero.key?(:post_id).must_equal false
+  end
+
+  it 'should have a Woman Representative post with an Area' do
+    ameede = memberships('mp_292').first
+    ameede[:post_id].must_equal 'woman_representative'
+  end
+
+  it 'should have a PWD post with no Area' do
+    ndeezi = memberships('mp_419').first
+    ndeezi[:post_id].must_equal 'pwd'
+  end
```

Comment:
Given the name of the test I guess this should be checking that the membership doesn't have an `area_id` key?